Pages/new_customer_page.py

```diff
@@ -33,3 +62,71 @@ def verify_message_by_input_name(self, input_name, expected_message):
             print(e)
 
         assert actual_message == expected_message, f'El mensaje en el campo {input_name} es {actual_message}. Se esperaba {expected_message}'
+    
+    def getID(self) -> str:
+        try:
+            element = self.driver.find_element(*NewCustomerLocators.id_customer_reg_msg)
+            return element.text
+        except Exception as e:
+            return e
+    
+    def write_customer_name(self, customer: str) -> None:
+        try:
+            self.driver.find_element(*NewCustomerLocators.customer_name_txt).send_keys(customer)
+        except Exception as e:
+            print(e)
+
+    def write_date_of_birth(self, date):
+        try:
+            self.driver.find_element(*NewCustomerLocators.date_birth_txt).send_keys(date)
+        except Exception as e:
+            print("Cannot write date")
+            print(e)
+
+    def write_address(self, address):
+        try:
+            self.driver.find_element(*NewCustomerLocators.address_txt).send_keys(address)
+        except Exception as e:
+            print(e)
+
+    def write_city(self, city: str) -> None:
+        try:
+            self.driver.find_element(*NewCustomerLocators.city_name_txt).send_keys(city)
+        except Exception as e:
+            print(e)
+
+    def write_state(self, state: str) -> None:
+        try:
+            self.driver.find_element(*NewCustomerLocators.state_name_txt).send_keys(state)
+        except Exception as e:
+            print(e)
+
+    def write_pin(self, pin: str) -> None:
+        try:
+            self.driver.find_element(*NewCustomerLocators.pin_name_txt).send_keys(pin)
+        except Exception as e:
+            print(e)
+
+    def write_phone(self, phone: str) -> None:
+        try:
+            self.driver.find_element(*NewCustomerLocators.phone_name_txt).send_keys(phone)
+        except Exception as e:
+            print(e)
+
+    def write_email(self) -> None:
+
+        length = 5
+        letters = string.ascii_lowercase
+        result_str = ''.join(random.choice(letters) for _ in range(length))
+        result_str += '@aol.com'
```

Comment:
You can return random email or move this logic on other method, something that allows us to know what is the email we provided for the costumer